### PR TITLE
PR: Update FontAwesome to 5.4.2

### PR DESCRIPTION
**Pull request by darkvertex**
Hey, it's me again. Putting this on the latest FA5 version. :tophat:

FYI the "regular" style did not add any new icons since the last update, but they did modify the fontfile so I put it as part of these changes. That's why there is nothing new in the "regular" charmap. (I believe they adjusted the look of an icon or two.)

This updates brings 52 new icons:
https://fontawesome.com/icons?d=gallery&s=brands&v=5.4.0,5.4.1,5.4.2&m=free
https://fontawesome.com/icons?d=gallery&s=solid&v=5.4.0,5.4.1,5.4.2&m=free

### CODE REVIEW

File: qtawesome/fonts/fontawesome5-brands-webfont-charmap.json [hunk 1]
```diff
@@ -271,7 +278,6 @@
     "reddit": "f1a1", 
     "reddit-alien": "f281", 
     "reddit-square": "f1a2", 
-    "rendact": "f3e4",
```
Comment (darkvertex) [inline qtawesome/fonts/fontawesome5-brands-webfont-charmap.json]: FYI this one didn't disappear, just got renamed to `wpressr`.

**Comment by darkvertex** [comment]
Hi @ccordoba12 did you release this in PyPI yet?

**Comment by darkvertex** [comment]
As I answer my own question, it seems you released it in PyPI but not this repo:
This is out: https://pypi.org/project/QtAwesome/0.5.3/
but in here there is no 0.5.3 as of writing this: https://github.com/spyder-ide/qtawesome/releases

That's why I got confused. You may wanna get those in sync.

**Comment by ccordoba12** [comment]
Sorry, I forgot to push the 0.5.3 tag, I just did it right now.

**Comment by darkvertex** [comment]
All good. Thanks!

**Pull request merged**